app/src/lib/status.ts

```diff
@@ -38,3 +39,9 @@ export function isConflictedFile(
 ): file is ConflictedFileStatus {
   return file.kind === AppFileStatusKind.Conflicted
 }
+
+export function hasConflictedFiles(
```

Comment:
Sorry for being difficult but seeing as this is an exported utility function could we please add a little bit of documentation here? It can be something simple like

```suggestion
/**
 * Returns a value indicating whether any of the files in the
 * working directory is in a conflicted state. See `isConflictedFile`
 * for the definition of a conflicted file.
 */
export function hasConflictedFiles(
```

Reply:
totally reasonable. thank you!